src/seedu/addressbook/data/AddressBook.java

```diff
@@ -54,6 +54,13 @@ public void removePerson(ReadOnlyPerson toRemove) throws PersonNotFoundException
         allPersons.remove(toRemove);
     }
 
+    /**
+     * Sorts the persons in the address book.
```

Comment:
It might be good to include *how* the persons are sorted e.g. Name, Birthday, Email etc.